src/modules/esl-carousel/core/esl-carousel.less

```diff
@@ -0,0 +1,58 @@
+// CORE styles
+esl-carousel {
+  position: relative;
+  display: block;
+  max-width: 100%;
+  // Important to ignore padding/margin hack
+  box-sizing: content-box;
+  margin: -5px;
+  padding: 5px;
+
+  overflow: hidden; // Fallback
+  /* stylelint-disable-next-line */
+  overflow: clip;
+
+  [data-slides-area] {
+    position: relative;
+    width: 100%;
+    display: flex;
+    gap: 20px;
```

Comment:
To make customization easier, maybe we should implement it through CSS variables?

Reply:
That one, I think, is really obvious to change. Moreover, it is not used anywhere else in CSS.
But the thing that we have a questionable default here, probably is not good... 
I've marked it for the follow-up to not mess around in the bounds of current PR.